collectors/pod_test.go

```diff
@@ -429,6 +431,26 @@ func TestPodCollector(t *testing.T) {
 					Status: v1.PodStatus{
 						HostIP: "1.1.1.1",
 						PodIP:  "2.3.4.5",
+						ContainerStatuses: []v1.ContainerStatus{
+							v1.ContainerStatus{
+								State: v1.ContainerState{
+									Terminated: &v1.ContainerStateTerminated{
+										FinishedAt: metav1.Time{
+											Time: time.Unix(1501769018, 0),
+										},
+									},
+								},
+							},
+							v1.ContainerStatus{
+								State: v1.ContainerState{
+									Terminated: &v1.ContainerStateTerminated{
+										FinishedAt: metav1.Time{
+											Time: time.Unix(1501779018, 0),
```

Comment:
Just to make sure the if condition works as intended, could you make this timestamp smaller than the first one (or add a third status with the middle one being that "latest")?